### PR TITLE
Fixes #10592 - Add tests to validate the allowed_helpers passed into the renderer in ProvisioningTemplate.build_pxe_default contains the generic list

### PR DESCRIPTION
**Pull request by imriz**
@domcleal can you please review?

### CODE REVIEW

File: test/unit/provisioning_template_test.rb [hunk 1]
```diff
@@ -209,5 +209,9 @@ class ProvisioningTemplateTest < ActiveSupport::TestCase
                                       :hostgroup_id => @hg3.id,
                                       :environment_id => @ev3.id}).name
     end
+    test "should call build_pxe_default with allowed_helpers containing the default helpers" do
```
Comment (dLobatog) [inline test/unit/provisioning_template_test.rb:213]: There should be an empty line in between tests

**Comment by dLobatog** [comment]
@imriz Thanks! I only left a nitpick comment inline, other than that I think the test is OK. Any reason you want @domcleal specifically to review it?

**Comment by imriz** [comment]
@dLobatog nope, I mentioned him just because he was commenting on the original bug, and I was chatting with him about it on IRC :)
Will fix the styling in just a sec.
Thanks for the review!

**Comment by imriz** [comment]
@dLobatog added the empty line as requested :)

**Comment by dLobatog** [comment]
Merged as 9f777254788160143231b329a85f0e9277642788, thanks @imriz!

**Pull request closed without merging**